src/__tests__/core/block.test.ts

```diff
@@ -15,6 +15,57 @@ test('block: select single block by number', async () => {
   expect(result).toEqual(expected);
 });
 
+test('block: select single transaction from block by index', async () => {
+  const query = `
+  {
+    block(number: 5771878) {
```

Comment:
@jonathanBuhler @akhila-raju (Just a nit) Generally I would suggest using lower block numbers in tests, the reason being that chain reorganisations are possible at any time, and the lower the block number the higher the guarantee that our test data will remain invariant.